pkg/sql/vtable/pg_catalog.go

```diff
@@ -1370,3 +1370,41 @@ CREATE TABLE pg_catalog.pg_subscription_rel (
 	srsublsn STRING,
 	srsubstate "char"
 )`
+
+//PgCatalogStatDatabase is an empty table in the pg_catalog that is not implemented yet
+const PgCatalogStatDatabase = `
+CREATE TABLE pg_catalog.pg_stat_database (
```

Comment:
can the columns be listed in the same order as they are here? https://www.postgresql.org/docs/9.2/monitoring-stats.html#PG-STAT-DATABASE-VIEW

Reply:
Sure, actually as a follow up, we might add the order of the column on the diff tool, right now, it is not taking care of the column order.

Reply:
Done.